drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/binary/downloading/source/GitHubSource.java

```diff
@@ -175,4 +222,14 @@ protected String getUniqueKey() {
     protected GitHubLastUpdateCache getCache() {
         return cache;
     }
+
+    // workaround for https://github.com/SpectoLabs/hoverfly/issues/573
+    protected String getRateLimitRemainingHeaderKey() {
```

Comment:
Is it really necessary to have workarounds in the production code for the tools we use in testing?